### DISCUSSION TITLE
请问出现这样的日志代表什么问题，可以正常输出pdf

### DISCUSSION THREAD

**Discussion by HanWuDi666** [note286/xduts#220] [Q&A] [answered]
![屏幕截图 2024-12-29 154659](https://github.com/user-attachments/assets/42c9dc9c-b4af-41c1-bfbd-992f26b9ee92)
![屏幕截图 2024-12-29 154634](https://github.com/user-attachments/assets/942ef7ec-100a-41af-8e88-1b5772dc7c1f)
![屏幕截图 2024-12-29 154732](https://github.com/user-attachments/assets/c770c7da-cb1f-4fb2-9695-fa7373b06aa4)

**Comment by note286** [comment, marked as answer]
这类警告无视即可，不是错误。

至于图中的`出现错误`字样意味着什么，并不清楚，建议你使用命令进行编译。

```
xelatex 论文初稿
```

并观察输出。
